lib/connection/statement.js

```diff
@@ -1334,6 +1335,10 @@ function sendRequestPreExec(statementContext, onResultAvailable)
     json.isInternal = statementContext.internal;
   }
 
+  if(!statementContext.disableQueryContextCache){
+    json.queryContextDTO = statementContext.services.sf.getQueryContextDTO();
```

Comment:
Shouldn't queryContext be set in function sendRequest in code below?

Reply:
I moved it.

Reply:
In my opinion I should in both places. Can you check it?

Reply:
I think we need this in both places. I added them in the both places.